Webapp/sources/services/queries/workgroup_request.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+
+from sources import models
+from sources.extensions import db
+
+
+def get_new_workgroup_requests() -> List[models.WorkGroupRequest]:
```

Comment:
I do think it's fine to group your services under domains, rather than continuing your database models/blueprints architecture into there. Like this would fit fine under the `workgroup` to me.

Reply:
Sounds good, I'll group the related ones